src/components/homepage/main-components/desktop-components/IntroDesktop.tsx

```diff
@@ -206,7 +206,7 @@ const IntroDesktop = () => {
                 type="email" 
                 name="" 
                 id="EMAIL" 
-                autoFocus
+                // autoFocus
```

Comment:
Remove it fully instead of commenting it out.

Reply:
Solved in #147